### PR TITLE
[build] Fix the `install-bazel.sh`

### PR DESCRIPTION
**Pull request by fishbone**
## Why are these changes needed?
install-bazel.sh is broken due to the path is not correctly set. This PR fixed it.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by simon-mo — APPROVED** [review]

File: ci/env/install-bazel.sh [hunk 1]
```diff
@@ -46,7 +46,7 @@ esac
 
 export PATH=/opt/python/cp36-cp36m/bin:$PATH
 python="$(command -v python3 || command -v python || echo python)"
-version="$("${python}" -s -c "import runpy, sys; runpy.run_path(sys.argv.pop(), run_name='__api__')" bazel_version "${ROOT_DIR}/../../python/setup.py")"
+version="$("${python}" -s -c "import runpy, sys; runpy.run_path(sys.argv.pop(), run_name='__api__')" bazel_version "${ROOT_DIR}/../python/setup.py")"
```
[the remark this answers is not in the document]
Comment (iycheng) [reply ci/env/install-bazel.sh:49]: Good catch. I think I know what's happening there now.

What I'm calling is `bash -x ./scripts/install-bazel.sh` and it a soft link to `ci/env/intall-bazel.sh` so the root dir was wrong. Let me see how to fix this.

File: ci/build/test-wheels.sh [hunk 2]
```diff
@@ -6,7 +6,7 @@ set -e
 # Show explicitly which commands are currently running.
 set -x
 
-ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
+ROOT_DIR=$(python -c "import os; print(os.path.dirname(os.path.realpath(r'${BASH_SOURCE:-$0}')));")
```
Comment (tupui) [inline ci/build/test-wheels.sh:9]: Not sure about going with Python there. We can follow the symlinks with bash as well and that would introduce minimal changes here.

```suggestion
ROOT_DIR=$(cd "$(dirname "$(readlink -f "${BASH_SOURCE:-$0}")"  )"; pwd)
```

Comment (tupui) [reply ci/build/test-wheels.sh:9]: On my side it is working as expected. But yes I've read that. In any case, I am not a core maintainer here so just suggesting things 😃

Comment (iycheng) [reply ci/build/test-wheels.sh:9]: @tupui No problem. Comments are always welcome and can help find a better way to support features.

I'm using a mac with 11.2.1 and it seems that 

```
readlink -f ./config.yaml
readlink: illegal option -- f
usage: readlink [-n] [file ...]
```

I feel if a newer version of mac can support this, maybe we can use this one. I don't like to introduce python as well and also it seems like windows can't be bootstrapped.  Let me workaround and also check the CI mac version.

Btw, really thank you for catching up on the issue introduced in this PR 😀

**Comment by iycheng** [comment]
Unfortunately, it's not working in CI :(

**Comment by iycheng** [comment]
test failure is not related.

**Pull request merged**